server/test/unit/base.py

```diff
@@ -270,5 +260,5 @@ def test_assert_equal_ignoring_list_order(self):
 class ResourceReservationTests(PulpServerTests):
     def tearDown(self):
         Worker.objects().delete()
-        ReservedResource.get_collection().remove()
+        ReservedResource.objects().delete()
```

Comment:
When chaining the `objects` call, calling it is not required and I think not calling `objects` makes it more readable. For example this line would become:  `ReservedResource.objects.delete()`. It's written using `objects()` in other places, but I would prefer those to be done differently too. Any rewrites of this in this PR should only deal with `ReservedResource.objects`.